user-management/src/main/scala/com/normation/plugins/usermanagement/UserManagementService.scala

```diff
@@ -145,10 +177,16 @@ object UserManagementService {
           (userXML \\ "authentication").head match {
             case e: Elem =>
               val newXml =
-                if (isPreHashed)
-                  e.copy(child = e.child ++ newUser.copy(password = newUser.password).toNode)
-                else
-                  e.copy(child = e.child ++ newUser.copy(password = getHash((userXML \\ "authentication" \ "@hash").text).encode(newUser.password)).toNode)
+                (isExtAuth, isPreHashed) match {
+                  case (true, _) =>
+                    authBackendsProvider match {
+                      case Some(extProvider) => e.copy(child = e.child ++ newUser.copy(password = extProvider).toNode)
```

Comment:
You need to put the password provided, it acts as a default one if external auth is out of service,   not the provider name

Reply:
But with next pr you will remove this